client/state/selectors/can-any-site-connect-domains.ts

```diff
@@ -0,0 +1,25 @@
+import { createSelector } from '@automattic/state-utils';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import getSites from 'calypso/state/selectors/get-sites';
+import type { AppState } from 'calypso/types';
+
+import 'calypso/state/ui/init';
+
+/**
+ * Return true if is user is able to use plugins on any of their sites.
```

Comment:
This comment is incorrect.

Reply:
Is this the gist of it?

```suggestion
 * Return true if a user has sites that are able to be connected to domains.
 * Returns false when a user has no sites or only self hosted Jetpack sites.
```

Reply:
Sorry, I forget to clean my inspiration 😉